examples/official-storybook/preview.js

```diff
@@ -87,7 +87,8 @@ const ThemedSetRoot = () => {
 };
 
 export const decorators = [
-  (StoryFn, { globals: { theme = 'light' } }) => {
+  (StoryFn, { globals, parameters }) => {
+    const theme = parameters.theme || globals.theme || 'light';
```

Comment:
This was added to assist with Chromatic and the `play` functionality -- The decorator in `examples/official-storybook` was causing chromatic to run each story 'stacked' with both light & dark themes. This made my play function invalid as it was only expecting a single element, but chromatic was rendering the story twice. This allows stories to override theme via parameters/global on a case-by-case basis. Hope this change is okay!

Reply:
Cool, I like it 👍

Reply:
Thank you! I ended up reverting this as it caused issues with the theming addon -- but something to keep in mind for future features 😄